lib/templates/netlifyFunction.js

```diff
@@ -19,6 +19,11 @@ const callbackHandler = (callback) =>
       ].map((value) => String(value));
     });
 
+    // NextJS by default sets Cache-Control to the max age. Without this
+    // override, the Netlify function will cache the very first page render.
+    // TO-DO: set this *only* for SSG'd pages
+    response.multiValueHeaders["cache-control"] = ["no-cache"];
```

Comment:
Hmmm... NextJS does not take care of this internally? You tested it?

If NextJS does not take care of it, then let's definitely implement this only for when preview mode cookies are set, so the user can still set a custom cache-control header for all other cases.

Reply:
Adding `cache-control: no-cache` seems to no longer be necessary and the code has been removed from the PR :heavy_check_mark: